src/context/redux/actions.js

```diff
@@ -0,0 +1,3 @@
+
+export const setOpen = () => ({ type: "OPEN" });
```

Comment:
actions can be simple objects that have a type. adding a wrapper function is unnecessary.

Reply:
removed wrapper function